pages/index.js

```diff
@@ -24,35 +26,44 @@ let ingredients = [
 ];
 
 export default function Home() {
+  const [ingredients, setIngredients] = useState(defaultIngredients);
+
   function editBagle(ingredient) {
-    if (ingredient.chosen) {
-      ingredient.chosen = false;
-    } else {
-      ingredient.chosen = true;
-    }
+    ingredient.chosen = !ingredient.chosen;
   }
 
   return (
     <div>
+      <Head>
+        <title>Parkbench Bagels</title>
+      </Head>
+
       <h1>Parkbench Bagels</h1>
 
       <h2>Build your own Bagle!</h2>
+      <form
+        onSubmit={(e) => {
+          // only for Testing
+          e.preventDefault();
+          console.log(e.t);
+        }}
+      >
+        <ul>
+          {ingredients.map((ingredient) => {
+            return (
+              <li key={ingredient.name}>
```

Comment:
`ingredient.name` as key is not unique enough (only for now as you use `defaultIngredients`). You should choose a unique identifier for your objects in an array.